apps/client/ui/InputSearch/InputSearch.tsx

```diff
@@ -3,15 +3,18 @@ import clsx from 'clsx';
 
 import SearchIcon from '@/ui/Icons/Search/Search';
 
-type InputSearchProps = ComponentPropsWithoutRef<'div'>;
+type InputSearchProps = ComponentPropsWithoutRef<'input'>;
```

Comment:
if you want to have native `input` props is better to spread them to `input` element below instead of `div`. but it's also good to keep `className` pointed to the root as it is now 👌